### ISSUE TITLE
invalid password error

### ISSUE THREAD

**Issue by opie546** [PeteRager/lennoxs30#45]
my password to login to https://www.lennoxicomfort.com/account/login contained an '&' and a '^' and I got errors stating my password was invalid. I moved the password from my secrets file to my config file and that did not fix anything. (tried restarting HA several times)

After verifying that I could login via a browser on my pc I changed my password (no special characters this time) and the custom component no longer had login issues

**Comment by PeteRager** [comment]
That looks like problem.

Did you try quoting the string?    ‘mypassword&^’

**Comment by opie546** [comment]
Yes I tried with and without quotes and it didn't make any difference

**Comment by PeteRager** [comment]
Thanks.  I will work on fixing this.

**Comment by PeteRager** [comment]
I have a solution for this, apologies for the delays, should have it integrated by end of next week.
